services/softlayer_network_storage.go

```diff
@@ -38,20 +38,33 @@ func (slns *softLayer_Network_Storage_Service) CreateIscsiVolume(size int, locat
 		return datatypes.SoftLayer_Network_Storage{}, errors.New("Cannot create negative sized volumes")
 	}
 
-	iscsiVolumeItemId, err := slns.getIscsiVolumeItemIdBasedOnSize(size)
-	if err != nil {
+	sizeItemPriceId, err := slns.getIscsiVolumeItemIdBasedOnSize(size)
+	iopsItemPriceId, err := slns.getPerformanceStorageItemPriceIdByIops(size)
+
+	/*if err != nil {
```

Comment:
Why is this line uncommented? I would expect a check for error after each call, so something like: 

```go
sizeItemPriceId, err := slns.getIscsiVolumeItemIdBasedOnSize(size)
if err != nil {
  return datatypes.SoftLayer_Network_Storage{}, err 
}

iopsItemPriceId, err := slns.getPerformanceStorageItemPriceIdByIops(size)
if err != nil {
  return datatypes.SoftLayer_Network_Storage{}, err 
}
```

Can you explain why you did not have check?